### DISCUSSION TITLE
"Please ensure that you are using the latest version" despite having the latest version

### DISCUSSION THREAD

**Discussion by marcostudios** [ppy/osu#27652] [Q&A]
![immagine](https://github.com/ppy/osu/assets/5209299/cd873b9d-210f-4467-8c07-2cf9b9d5b5ad)

![immagine](https://github.com/ppy/osu/assets/5209299/1149a463-7789-43be-867b-f8cfb9dca0e6)

It shows that error but i have the latest version downloaded from the osu! website and updated when a new release happens

Logs: [compressed-logs.zip](https://github.com/ppy/osu/files/14628137/compressed-logs.zip)

**Comment by SupDos** [comment]
```
2024-03-17 16:36:43 [verbose]: Performing request osu.Game.Online.Solo.CreateSoloScoreRequest
2024-03-17 16:36:44 [verbose]: Request to https://osu.ppy.sh/api/v2/beatmaps/2069131/solo/scores failed with System.Net.WebException: UnprocessableEntity.
2024-03-17 16:36:44 [verbose]: Response was: {"error":"missing token header"}
```

Reading https://github.com/ppy/osu/issues/27290 which seems to be related, apparently deleting `%APPDATA%\osu\.require_update` fixed it for most people in that issue. Is that file there for you? Maybe try that

**Reply by marcostudios** [reply]
I don't have that file. It probably got removed when i closed osu because it was dinner time. The issue is solved now, thanks for answering.

![immagine](https://github.com/ppy/osu/assets/5209299/6117a291-d45f-4020-95d1-6ca9e17bdd32)

**Reply by marcostudios** [reply]
Sorry, i could play perfectly last night but now the issue has returned. And i don't have the .require_update on that folder

![immagine](https://github.com/ppy/osu/assets/5209299/46b47b83-554c-4d81-b1a0-49e3cedf07d9)

**Comment by catarium** [comment]
Have the same issue.
It works some days but not others.
Nothing seems to help (deleting .require_update and AuthNative.dll, deleting and reinstalling lazer completely, running from other windows user)
No problems with the version built from source code (on test servers) and stable.
